src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerappjob_registry.py

```diff
@@ -0,0 +1,69 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+
+from msrestazure.tools import parse_resource_id
+
+from azure.cli.testsdk.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer, JMESPathCheck, JMESPathCheckExists,
+                               JMESPathCheckNotExists)
+
+from azure.cli.command_modules.containerapp.tests.latest.common import TEST_LOCATION
+from .utils import prepare_containerapp_env_for_app_e2e_tests
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+# flake8: noqa
+# noqa
+# pylint: skip-file
+
+
+class ContainerAppJobsRegistryOperationsTest(ScenarioTest):
+    # def __init__(self, *arg, **kwargs):
+    #     super().__init__(*arg, random_config_dir=True, **kwargs)
+
+    @ResourceGroupPreparer(location="northeurope")
+    @AllowLargeResponse()
+    def test_containerapp_registry_curd(self, resource_group):
```

Comment:
should this be `test_containerapp_registry_crud`?

Reply:
Fixed.